packages/ember/index.js

```diff
@@ -202,7 +202,20 @@ Ember.canInvoke = utils.canInvoke;
 Ember.tryInvoke = utils.tryInvoke;
 Ember.wrap = utils.wrap;
 Ember.uuid = utils.uuid;
-Ember.NAME_KEY = utils.NAME_KEY;
+Ember.setName = utils.setName;
```

Comment:
if we prefer toString on the constructor, why are we exporting this?

Reply:
Derp. I missed this when I last reviewed. Good catch @krisselden.

Reply:
remove this and it is good to go